enhancements/audit-command.md

```diff
@@ -0,0 +1,722 @@
+---
+title: audit-command-operation
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-22
+last-updated: 2021-03-02
+status: implementable
+see-also:
+  - [catalog builds in operator-sdk](operator-sdk-builds-catalogs.md)
+  - [custom operator validation #43](https://github.com/operator-framework/enhancements/pull/43)
+  - [Enhancement for validating bundles in a catsrc image #48](https://github.com/operator-framework/enhancements/pull/48)
+  - [operator-framework/api: add new checks to the operatorhub validator][ep-new-checks for-operator-hub]
+---
+
+# Audit command operation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions 
+
+1. Should this command to be added to SDK or OPM or in a new tool?
+
+**Context**
+
+The audit command will perform checks that are implemented and are provided via SDK tool commands `bundle validate` and `scorecard` already. Then, it could be done from SDK binary or use it. However, the OPM tool is the tool that has been used with the purpose of admin the catalogs. 
```

Comment:
It seems weird for `opm` to call `operator-sdk` binary. And equally weird to have `audit` command added to `operator-sdk` binary. Seems to me that if `opm` needs access to the things that `bundle validate` and `scorecard` need to do we should look at pulling that functionality out into a consumable library that could be used by both tools.

Reply:
The EP does not define where it would be done. However, it might not be done in SDK or OPM either.  Please check that all these points are better clarified now in the EP as well.